documentation/staging/content/samples/simple/domains/fmw-domain-home-in-image/_index.md

```diff
@@ -0,0 +1,475 @@
+  namespace as the domain.
+* Unless you are creating a Restricted-JRF domain, you would also need to:
+  * Configure access to your database. For details, see [here]({{< relref "/userguide/managing-fmw-domains/fmw-infra/_index.md#configuring-access-to-your-database" >}}).  
+  * Create a Kubernetes Secret with the RCU credentials. For details, refer to this [document](https://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-rcu-credentials/README.md).
+
+#### Use the script to create a domain
+
+The sample for creating domains is in this directory:
+
+```shell
+$ cd kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image
+```
+Make a copy of the `create-domain-inputs.yaml` file, update it with the correct values. 
+If `fwmDomainType` is `JRF`, also update the input files with configurations for accessing the RCU database schema,
+including `rcuSchemaPrefix`, `rcuSchemaPassword`, `rcuDatabaseURL`, and `rcuCredentialSecrets`.
+Run the create script, pointing it at your inputs file and an output directory, along with username and password for the WebLogic administrator::
+
+```shell
+$ ./create-domain.sh \
+  -u <username> \
+  -p <password> \
+  -i create-domain-inputs.yaml \
+  -o /<path to output-directory>
+```
+
+The script will perform the following steps:
+
+* Create a directory for the generated Kubernetes YAML files for this domain if it does not
+  already exist.  The path name is `/<path to output-directory>/weblogic-domains/<domainUID>`.
+  If the directory already exists, its contents must be removed before using this script.
+* Create a properties file, `domain.properties`, in the directory that is created above. This properties file will be used to create a sample FMW Infrastructure domain.
+* Download the latest [WebLogic Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT) and [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT) installer ZIP files to your `/tmp/dhii-sample/tools` directory. Both WDT and WIT are required to create your Model in Image container images.
+  Visit the GitHub [WebLogic Deploy Tooling Releases](https://github.com/oracle/weblogic-deploy-tooling/releases) and [WebLogic Image Tool Releases](https://github.com/oracle/weblogic-image-tool/releases) web pages to determine the latest release version for each.
+
+* Set up the WebLogic Image Tool to the `/tmp/dhii-sample/tools/imagetool` directory. Set the
+  WIT cache store location to the `/tmp/dhii-sample/tools/imagetool-cache` directory and
+  put a `wdt_<WDT_VERSION>` entry in the tool's cache which points to the path of the WDT ZIP file installer.
+  For more information about the WIT cache, see the
+  [WIT Cache documentation](https://github.com/oracle/weblogic-image-tool/blob/master/site/cache.md).
+
+* Invoke the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT) to create a new 
```

Comment:
Apply this advice to this document also: You don't have to keep repeating the acronym. After its first introduction, either use the tool name or the acronym, but not both. Please fix this globally.